lib/reporters/TapReporter.js

```diff
@@ -46,12 +46,14 @@ export default class TapReporter {
     console.log(`  message: "${error.message || 'failed'}"`)
     console.log(`  severity: ${severity || 'failed'}`)
 
-    if (error.actual) {
-      console.log(`  actual: ${JSON.stringify(error.actual, null, 2)}`)
+    if (error.hasOwnProperty('actual')) {
+      var actualStr = error.actual ? JSON.stringify(error.actual, null, 2) : 'undefined'
```

Comment:
This check should be stricter, otherwise we'll get `undefined` for any "falsy" value.